File: 백준/Bronze/30468. 호반우가 학교에 지각한 이유 1/호반우가 학교에 지각한 이유 1.cs
 string[] inputs = Console.ReadLine().Split();
int STR = int.Parse(inputs[0]);
int DEX = int.Parse(inputs[1]);
int INT = int.Parse(inputs[2]);
int LUK = int.Parse(inputs[3]);
int N = int.Parse(inputs[4]);

int totalStatus = STR + DEX + INT + LUK;

if (totalStatus < 4 * N)
{
    Console.WriteLine((4 * N) - totalStatus);
}
else
{
    Console.WriteLine(0);
}

File: 백준/Bronze/30468. 호반우가 학교에 지각한 이유 1/호반우가 학교에 지각한 이유 1.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int STR = sc.nextInt();
        int DEX = sc.nextInt();
        int INT = sc.nextInt();
        int LUK = sc.nextInt();
        int N = sc.nextInt();

        int totalStatus = STR + DEX + INT + LUK;

        if (totalStatus < 4 * N) {
            System.out.println((4 * N) - totalStatus);
        } else {
            System.out.println(0);
        }
    }
}
